setup_toolchains.py

```diff
@@ -3,41 +3,61 @@
 
 _AVAILABLE_TOOLCHAINS = {
     'x86-64': {
-        '2021.11-5': {
-            'sha256': '6fe812add925493ea0841365f1fb7ca17fd9224bab61a731063f7f12f3a621b0',
-            'platform_arch': 'x86_64',
-            'tool_prefix': 'x86_64-buildroot-linux-gnu',
-        },
-        '2022.08-1': {
-            'sha256': '861c1e8ad0a66e4c28e7a1f8319d68080ab0ff8d16a765e65540f1957203a190',
-            'platform_arch': 'x86_64',
-            'tool_prefix': 'x86_64-buildroot-linux-gnu',
+        'gnu': {
+          '2021.11-5': {
+              'sha256': '6fe812add925493ea0841365f1fb7ca17fd9224bab61a731063f7f12f3a621b0',
+              'platform_arch': 'x86_64',
```

Comment:
Doesn't this need a 'cstdlib' entry?  How did this pass the tests?

Reply:
Its under a cstdlib right? The GNU above?

Reply:
Done